package-lock.json

```diff
@@ -3229,7 +3229,7 @@
     "chainsaw": {
       "version": "0.1.0",
       "resolved": "https://registry.npmjs.org/chainsaw/-/chainsaw-0.1.0.tgz",
-      "integrity": "sha512-75kWfWt6MEKNC8xYXIdRpDehRYY/tNSgwKaJq+dbbDcxORuVrrQ+SEHoWsniVn9XPYfP4gmdWIeDk/4YNp1rNQ==",
+      "integrity": "sha1-XqtQsor+WAdNDVgpE4iCi15fvJg=",
```

Comment:
These dependencies are going from using SHA512 to SHA1, and I believe SHA512 is the more secure hash function.  Do we need to rerun `npm i` here after deleting the node_modules folder possibly?

Reply:
Good catch, I did run a clean and install, but I'll try again just to be sure.